avm/res/data-protection/backup-vault/backup-instance/main.bicep

```diff
@@ -0,0 +1,98 @@
+metadata name = 'Data Protection Backup Vault Backup Instances'
+metadata description = 'This module deploys a Data Protection Backup Vault Backup Instance.'
+
+@description('Conditional. The name of the parent Backup Vault. Required if the template is used in a standalone deployment.')
+param backupVaultName string
+
+@description('Required. The name of the backup instance.')
+param name string
+
+@description('Optional. The friendly name of the backup instance.')
+param friendlyName string?
+
+@description('Required. Gets or sets the data source information.')
+param dataSourceInfo dataSourceInfoType
+
+@description('Required. Gets or sets the policy information.')
+param policyInfo policyInfoType
+
+resource backupVault 'Microsoft.DataProtection/backupVaults@2023-05-01' existing = {
+  name: backupVaultName
+
+  resource backupPolicy 'backupPolicies@2023-05-01' existing = {
+    name: policyInfo.policyName
+  }
+}
+
+var policyInfoVar = {
```

Comment:
Why an extra variable if its only used in one location?

Reply:
removed